lab-elizabeth/app/view/home/home.html

```diff
@@ -0,0 +1,18 @@
+<main class="home">
+
+  <create-gallery></create-gallery>
+
+  <nav class="gallery-list-container">
+
+    <ul>
+      
+      <gallery-li
+        ng-repeat="item in homeCtrl.galleries"
+        gallery="item">
```

Comment:
Good use of ng-repeat and also passing the specific gallery to the scope of the galleryLIController using data binding :)